mws/resource_mws_workspaces.go

```diff
@@ -577,7 +583,7 @@ func ResourceMwsWorkspaces() common.Resource {
 			var workspace Workspace
 			workspacesAPI := NewWorkspacesAPI(ctx, c)
 			common.DataToStructPointer(d, workspaceSchema, &workspace)
-			if err := requireFields(c.IsAws(), d, "aws_region", "credentials_id", "storage_configuration_id"); err != nil {
```

Comment:
Should we have two cases - when compute_mode is serverless, then use this new condition, and when isn't, then use the old condition?